docker/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:lts-alpine AS build
+WORKDIR /pack
+COPY --chown=1000 . .
+RUN npm pack
+
+FROM node:lts-alpine AS prod
+# hadolint ignore=DL3010
+COPY --from=build /pack/markdownlint-cli2-*.tgz /
+RUN npm install --global --no-package-lock --production markdownlint-cli2-*.tgz && \
+    rm /markdownlint-cli2-*.tgz
```

Comment:
Honestly, I hadn't considered the impact on the image size. My only thought was to cleanup after the operation completed.

Now that I think about it, since the `rm` operation is performed as part of the same `RUN` command as the `npm install`, it will not add another layer. Each _docker_ command (COPY, ADD, RUN, USER, etc...)  in a Dockerfile adds a new layer. But multiple commands executed as part of a single `RUN` command are all included in the same layer.

Which means that in theory, the final image size should be smaller, because this layer adds less bytes to the filesystem. In practice, however, because the `markdownlint-cli2-0.2.0.tgz` file is only about 12KB, its inclusion/exclusion makes no perceivable difference to the image size.

So I think I should leave this as is, for cleanliness and to follow best practices. Agreed?